### DISCUSSION TITLE
Ship webview2 in the binary

### DISCUSSION THREAD

**Discussion by heyrict** [DioxusLabs/dioxus#1256] [General]
I'm writing a gui application, which targets to various version of Windows including aged ones e.g. windows 7 and 8 systems.
Unfortunately, it is hard to install webview2 on these outdated OS [due to Microsoft ending support for them](https://blogs.windows.com/msedgedev/2022/12/09/microsoft-edge-and-webview2-ending-support-for-windows-7-and-windows-8-8-1/).
Afaik [tauri provides an option to ship a fixed version of webview2](https://tauri.app/v1/guides/building/windows/#fixed-version), can we similarly ship webview2 in the dioxus binary?

**Comment by ealmloff** [comment]
There is some discussion of this issue here https://github.com/DioxusLabs/dioxus/issues/87

**Comment by heyrict** [comment]
Thank you for the reference! I think this issue should be resolved by the recently introduced dioxus bundler in v0.4. Thank you for your great work!
